content/Live Streaming Apps Architecture - Part 1 - German Lopez.md

```diff
@@ -0,0 +1,182 @@
+<img src="images/live-streaming-architecture.png" alt="Live Streaming Architecturing"/>
+
+# Live Streaming apps architecture - Part 1
+
+**Abstract:** A few ideas and considerations for architecting a live streaming application based on our previous experience.
+
+## Live streaming
+
+The term implies that the media(audio, video or both) is broadcasted to the end users in real time. This is different than just _streaming_ which could refer to previously recorded content such as VOD(Video On Demand), music streaming, etc. The later means that the media is delivered and played on demand rather than being generated simoultaneously.
```

Comment:
```suggestion
The term implies that the media(audio, video, or both) is broadcasted to the end-users in real-time. This is different than just _streaming_ which could refer to previously recorded content such as VOD(Video On Demand), music streaming, etc. Streaming media is delivered and played on demand rather than being generated simultaneously.
```